addons/web/static/tests/views/fields/many2many_tags_avatar_field_tests.js

```diff
@@ -445,7 +446,7 @@ QUnit.module("Fields", (hooks) => {
         );
         assert.strictEqual(
             o_kanban_record.querySelector("img.o_m2m_avatar").dataset.src,
-            "/web/image/partner/4/avatar_128",
+            `${getOrigin()}/web/image/partner/4/avatar_128`,
```

Comment:
prefer full assert rather than includes (I want to see the unique key in all tests)